.github/workflows/ci.yml

```diff
@@ -104,17 +104,17 @@ jobs:
         uses: dtolnay/rust-toolchain@stable
         id: install-rust
         with:
-          toolchain: '1.84.1'
-          targets: 'wasm32-wasip1, wasm32-unknown-unknown'
-          components: 'llvm-tools-preview, rustfmt, clippy'
```

Comment:
I think our team is somehow using different formatters on `yml` file. Will be good to add a format check before commits and in the CI